### PR TITLE
update cloud quickstart

### PR DESCRIPTION
**Pull request by micheleRP**
fixes https://github.com/redpanda-data/documentation-private/issues/2036

preview: https://deploy-preview-78--redpanda-docs-preview.netlify.app/current/get-started/quick-start-cloud/

### CODE REVIEW

File: modules/get-started/pages/quick-start-cloud.adoc [hunk 1]
```diff
@@ -25,41 +23,21 @@ To create a cluster:
 
 . Click **Create Dedicated cluster**, and enter the following values:
 +
-- **Cluster name**:
+- **Cluster name**: `redpandaquickstart`
```
Comment (JakeSCahill) [inline modules/get-started/pages/quick-start-cloud.adoc]: I don't think we need the monospace font for user input in the UI.

File: modules/get-started/pages/quick-start-cloud.adoc [hunk 2]
```diff
@@ -100,27 +78,23 @@ export REDPANDA_SASL_MECHANISM="SCRAM-SHA-256"
...
 +
 - **Host**: *
-- **Topic name**:
-+
-[.no-copy]
-----
-chat-room
-----
-
+- **Topic name**: `chat-room`
```
Comment (JakeSCahill) [inline modules/get-started/pages/quick-start-cloud.adoc:75]: Same comment as above

**Review by JakeSCahill — APPROVED** [review]
Looks good. Just a comment about monospace text for user input.

File: modules/get-started/pages/quick-start-cloud.adoc (hunk 1 above)
Comment (micheleRP) [reply modules/get-started/pages/quick-start-cloud.adoc]: Right, thanks @JakeSCahill! I changed user inputs from monospace to bold.

**Pull request merged**